### PR TITLE
ACMS-2008: Update scripts and installation hooks to gin theme as admin theme.

### PR DESCRIPTION
**Pull request by vishalkhode1**
**Motivation**
Update scripts and installation hooks to gin theme as admin theme.

**Proposed changes**
-  Removed all references for acquia_claro.
- Included gin modules/theme dependencies.

**Alternatives considered**
N/A

**Testing steps**
- Run command `./vendor/bin/acms site:install --yes` and site installation should work as expected, setting gin theme & enable gin modules.

**Merge requirements**
- [ ] _Minor change_
- [ ] Manual testing by a reviewer

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,15 +25,18 @@
...
         "drupal/consumer_image_styles": "^4.0",
+        "drupal/gin_toolbar": "^1.0@RC",
         "drupal/google_analytics": "^4.0",
         "drupal/google_tag": "^2.0",
         "drupal/honeypot": "^2.1",
         "drupal/recaptcha": "^3.2",
         "drupal/reroute_email": "^2.2",
         "drupal/shield": "^1.7",
         "drupal/sitestudio_config_management": "dev-develop",
+        "drupal/sitestudio_gin": "^1.0",
```
Comment (rajeshreeputra) [inline composer.json:39]: This is not required.

File: composer.json [hunk 2]
```diff
@@ -25,15 +25,18 @@
         "drupal/acquia_cms_site_studio": "dev-develop",
         "drupal/acquia_cms_starter": "dev-develop",
         "drupal/acquia_cms_toolbar": "dev-develop",
+        "drupal/acquia_cms_toolbar_gin": "^1.0",
```
Comment (rajeshreeputra) [inline composer.json:28]: This is not required.

File: composer.json [hunk 3]
```diff
@@ -25,15 +25,18 @@
...
         "drupal/acquia_cms_tour": "dev-develop",
         "drupal/consumer_image_styles": "^4.0",
+        "drupal/gin_toolbar": "^1.0@RC",
```
Comment (rajeshreeputra) [inline composer.json:31]: Instead of gin_toolbar module add gin theme as it's dependency of Gin theme.
```suggestion
        "drupal/gin": "^3.0",
```

**Review by rajeshreeputra — DISMISSED** [review]
Requested minor changes, please take a look.

**Review by chandan-singh7929 — APPROVED** [review]

**Pull request merged**